packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -2369,6 +2369,28 @@ describe('network stubbing', { retries: { runMode: 2, openMode: 0 } }, function
         .wait('@get').its('response.body').should('deep.eq', '{ "foo": "bar" }')
       })
 
+      // @see https://github.com/cypress-io/cypress/issues/16722
```

Comment:
@flotwig can you enlighten me about the `@see` annotation? I see it used in many places. I have read https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#adding-links-within-code but it doesn't mention what this does/means.

Reply:
`@see` is part of jsdoc, you can read about the see tag here: https://jsdoc.app/tags-see.html

In general we want a link in the comments to the relevant GitHub issue that the test or code is addressing for our team's future context.